packages/cli/src/util/dev/server.ts

```diff
@@ -1566,9 +1567,31 @@ export default class DevServer {
 
             // Retain orginal pathname, but override query parameters from the rewrite
```

Comment:
This comment is the same on line 1589. Is that expected?

Reply:
Nope! Removed in [`9a511e1` (#8457)](https://github.com/vercel/vercel/pull/8457/commits/9a511e1191a6bdfe9fd312bc1655765557590ec0)